client/my-sites/domains/domain-management/name-servers/index.jsx

```diff
@@ -70,31 +71,57 @@ class NameServers extends React.Component {
 		return this.props.isRequestingSiteDomains || ! this.props.nameservers.hasLoadedFromServer;
 	}
 
-	render() {
-		const classes = classNames( 'name-servers', {
-			'is-placeholder': this.isLoading(),
-		} );
+	isPendingTransfer() {
+		const domain = getSelectedDomain( this.props );
 
-		return (
-			<Main className={ classes }>
-				{ this.header() }
+		return get( domain, 'pendingTransfer', false );
+	}
+
+	getContent() {
+		const domain = getSelectedDomain( this.props );
 
+		return (
+			<React.Fragment>
```

Comment:
TIL `React.Fragment`!

Reply:
It's newish. React 16.2: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html